Source/JavaScriptCore/runtime/JSONObject.cpp

```diff
@@ -1021,17 +1022,18 @@ void FastStringifier::append(JSValue value)
             recordBufferFull();
             return;
         }
-        m_buffer[m_length] = '"';
+        auto* cursor = m_buffer + m_length;
+        *cursor++ = '"';
```

Comment:
Does this set of changes, to use pointers instead of indexing, improve performance? I ask because many years ago I often changed array indexing to pointer manipulation thinking I was making code faster, and others pointed out many cases where it was performance neutral and just code churn. I can readily believe, though, that here it might be an effective optimization, if you checked or measured it.

Reply:
Yeah, interestingly, this steadily contributes to 2% of the measured improvement.